apps/aechannel/src/aesc_client_reconnect_tx.erl

```diff
@@ -124,40 +128,48 @@ signers(#channel_client_reconnect_tx{role = Role} = Tx, Trees) ->
 -spec serialize(tx()) -> {vsn(), list()}.
 serialize(#channel_client_reconnect_tx{
              channel_id = ChannelId
+           , round      = Round
            , role       = Role
            , pub_key    = Pubkey }) ->
     {?INITIAL_VSN,
      [ {channel_id, ChannelId}
+     , {round     , Round}
      , {role      , atom_to_binary(Role, utf8)}
      , {pub_key   , Pubkey}
      ]}.
 
 -spec deserialize(vsn(), list()) -> tx().
 deserialize(?INITIAL_VSN, [ {channel_id, ChannelId}
+                          , {round     , Round}
                           , {role      , Role0}
                           , {pub_key   , Pubkey} ]) ->
     channel = aeser_id:specialize_type(ChannelId),
     account = aeser_id:specialize_type(Pubkey),
+    true = is_integer(Round) andalso Round > 0,
```

Comment:
Type `non_neg_integer()` but checked for positive? 🤔

Reply:
Well, the type info is just informative. :)